smart-hungry-delete.el

```diff
@@ -124,6 +130,14 @@ completely deleted."
   (add-hook 'nxml-mode-hook 'smart-hungry-delete-default-sgml-mode-common-hook)
   (add-hook 'text-mode-hook 'smart-hungry-delete-default-text-mode-hook))
 
+;;;###autoload
+(defun smart-hungry-delete-should-add-kill-ring (should-kill)
```

Comment:
common elisp convention is to postfix predicate functions with `-p`: e.g. `-any-p`, `ring-p`, `org-url-p`.

How about `smart-hungry-delete-kill-ring-save-p`?

Reply:
Wait, this isn't a predicate. Hmm.